src/lib/components/event/event-category-filter.svelte

```diff
@@ -32,7 +32,8 @@
       value: _value,
       url: $page.url,
     }).then((v) => {
-      _value = v?.toString() ?? null;
+      const hasOption = options.find(({ option }) => option === v);
+      _value = hasOption ? v?.toString() ?? null : null;
```

Comment:
Slightly confusing here... Why not use hasOption.toString()?

Reply:
Definitely not ideal with that nested logic 😅 Updated to 👇

```js
      const visibleOption = options.find(({ option }) => option === v);
      _value = visibleOption?.option?.toString() ?? null;
```